.github/workflows/benchmarks.yml

```diff
@@ -97,23 +97,23 @@ jobs:
     strategy:
       matrix:
         target:
-          - platform: "linux"
-            architecture: "x86_64"
+          - family: "linux"
```

Comment:
IMHO, `family` is a bit confusing. Reading from L100 to L106, it looks like linux-rv64 is not part of the linux family.  `arch` and `platform` have a list of supported options at `target_platform.h`
https://github.com/iree-org/iree/blob/1c881f57470564c55c8087d1772b4b8797b058f9/runtime/src/iree/base/target_platform.h#L13-L55

Maybe we should align the usage.

Reply:
Yeah, agree. The `family` is introduced just for selecting the `build_*.sh` script and populating `*_ARCH` and *_ABI` environment variables.

IMO there are two ways to do this:
1. Drop the `family` and replace `*_ARCH` and `*_ABI` with `BUILD_ARCH` and `BUILD_ABI` (originally we chose `*_` names to allow users to set ARCH/ABI for different scripts at the same time. But I suspect it's uncommon to run these scripts outside CI)
2. Have `build_linux.sh` script to call `build_riscv.sh` to do cross-compile when the ARCH is rv32/64. So `linux-rv32/64` can join the `linux family`. Basically we always call `build_${platform}.sh` and let it call proper scripts to cross-compile.

Personally I prefer 1. because of its simplicity. And IMO 2. adds more logic in the scripts, which makes them harder to understand.

Reply:
I am okay with 1). 2) will be too convolved for cross-compile target. Thoughts, @ScottTodd ?

Reply:
Note that "platform" isn't quite "os", though, since it can include things like emscripten. Being consistent with target_platform.h makes sense to me.